client/sandbox/react-nextjs/pages/play/typed-presence.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Head from 'next/head';
+import { i, init, RoomHandle } from '@instantdb/core';
```

Comment:
We use `RoomHandle` here, do we also need to export it from `@instantdb/react` and friends, or are not expecting users to use `RoomHandle`?